feature_apps/src/main/java/cm/aptoide/pt/feature_apps/data/network/analytics/IAnalyticsInfoProvider.kt

```diff
@@ -0,0 +1,6 @@
+package cm.aptoide.pt.feature_apps.data.network.analytics
+
+interface IAnalyticsInfoProvider {
```

Comment:
We don't name interfaces with this prefix. Use just `AnalyticsInfoProvider`
Then implementations can have some prefix like `AptoideAnalyticsInfoProvider`